### PR TITLE
feat(standups): Add slack configuration link to standup options

### PR DESCRIPTION
**Pull request by jmtaber129**
# Description
See [slack discussion](https://parabol.slack.com/archives/C042HTVBH8D/p1695159030945889) 🔒 

Add a standup meeting option menu item for "slack configuration" that links to the team's integrations page.

Also add metrics for when this option is clicked (note: this won't fire if the user specifically opens the link in a new tab instead of clicking)

## Demo
Screen Shot 2023-09-21 at 1 45 40 PM

## Testing scenarios
- [ ] UI makes sense
- [ ] Clicking sends the user to integration options in a new tab and sends a metric
- [ ] User can open the link by manually opening in new tab

## Final checklist

- [X] I checked the [code review guidelines](../docs/codeReview.md)
- [X] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [X] I have performed a self-review of my code, the same way I'd do it for any other team member
- [X] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [X] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [X] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [X] PR title is human readable and could be used in changelog

### CODE REVIEW

**Review by tghanken — APPROVED** [review]
Analytics looks good!

File: packages/client/components/TeamPrompt/TeamPromptOptionsMenu.tsx [hunk 1]
```diff
@@ -130,6 +132,28 @@ const TeamPromptOptionsMenu = (props: Props) => {
...
+        label={
+          <Link
+            to={`/team/${team.id}/settings/integrations`}
+            target='_blank'
+            rel='noopener noreferrer'
+          >
+            <OptionMenuItem>
+              <SlackSVG />
+              <span className='ml-2'>Configure slack</span>
+              <OpenInNew className='ml-1 text-base text-slate-600' />
```
Comment (BartoszJarocki) [inline packages/client/components/TeamPrompt/TeamPromptOptionsMenu.tsx:146]: +1 imo it'd be better to push it to the right, looks a little bit cleaner 
Screenshot 2023-09-21 at 12 30 35

File: packages/client/components/TeamPrompt/TeamPromptOptionsMenu.tsx [hunk 2]
```diff
@@ -130,6 +132,28 @@ const TeamPromptOptionsMenu = (props: Props) => {
           openRecurrenceSettingsModal()
         }}
       />
+      <MenuItem
+        key='slack'
+        label={
+          <Link
+            to={`/team/${team.id}/settings/integrations`}
+            target='_blank'
+            rel='noopener noreferrer'
```
Comment (BartoszJarocki) [inline packages/client/components/TeamPrompt/TeamPromptOptionsMenu.tsx:141]: +1 do we actually want to use a new tab? it's within our app, imo it'd be better to just use a normal link

File: packages/client/components/TeamPrompt/TeamPromptOptionsMenu.tsx [hunk 3]
```diff
@@ -130,6 +132,28 @@ const TeamPromptOptionsMenu = (props: Props) => {
...
+          >
+            <OptionMenuItem>
+              <SlackSVG />
+              <span className='ml-2'>Configure slack</span>
```
Comment (BartoszJarocki) [inline packages/client/components/TeamPrompt/TeamPromptOptionsMenu.tsx:145]: +1 `Slack` instead of `slack`?

**Review by BartoszJarocki — APPROVED** [review]
looks good! left some comments to consider

File: packages/client/components/TeamPrompt/TeamPromptOptionsMenu.tsx (hunk 2 above)
Comment (jmtaber129) [reply packages/client/components/TeamPrompt/TeamPromptOptionsMenu.tsx:141]: The thinking here is that even though it's within the app, it's still outside of the meeting, so I'm wary of taking the user completely out of the meeting potentially unexpectedly.

Comment (enriquesanchez) [reply packages/client/components/TeamPrompt/TeamPromptOptionsMenu.tsx:141]: 🤔 If the user has added content but not clicked Submit, will they lose their work if they are taken to the Slack config page?

**Review by enriquesanchez — APPROVED** [review]
LGTM! 👍

**Pull request merged**